rust_sodium-sys/build.rs

```diff
@@ -394,3 +557,14 @@ fn main() {
     println!("cargo:rustc-link-search=native={}/lib", install_dir);
     println!("cargo:include={}/include", install_dir);
 }
+
+#[cfg(all(not(windows), not(feature = "use-installed-libsodium")))]
+fn main() {
+    use std::env;
+    let target = unwrap!(env::var("TARGET"));
+    if target.contains("-ios") {
+        main_ios();
+    } else {
+        main_non_ios();
+    }
```

Comment:
I tried to do this check with conditional compilation (`#[cfg(target_family = "ios")]`), but for some reason that did not seem to work... And maybe it's better if the code is compiled in unconditionally (on non-windows at least), to be able to catch compile errors even when not compiling for an iOS target.